### DISCUSSION TITLE
Matomo - an alternative for Google Analytics?

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#3486] [General]
Although GA is an important tool, in the last year I have received requests to find other safe alternatives that protect personal data. What I have analyzed so far, Matomo seems to be a good option (with several paid plugins). The advantage is that all data are stored in a local database (MySQL) and the whole solution is implemented in PHP (version 4 >7.2).

In addition, there is an extension for Magento 1 https://github.com/vianetz/matomo-magento1 (last update 3 months ago). 

If anyone has faced this problem, please leave a feedback.

**Comment by fballiano** [comment]
I've added that extension to https://github.com/fballiano/awesome-openmage

**Reply by addison74** [reply]
All right. I haven't tested it, but it can be useful to those who would like to give Matomo a try. So far I am satisfied with what I see, but I have to pay for some plugins. Saving data locally is worth the price. I inserted the code manually according to the instructions. on YouTube there is a playlist created by the one who founded Matomo, a Frenchman, very well structured.
